content/blog/2025/06/2025-06-11-announcing-premium-tiers.md

```diff
@@ -0,0 +1,63 @@
+
+However, we can’t continue to bear the cost of the server as is, and before we get to the extreme position of being forced to turn it off leaving its 370k monthly active users in the awkward position of finding a new home for their account, we’ve decided to try to alleviate some of these costs by setting-up a freemium offering and proposing premium plans in addition to the free ones. The goal is to get the server to an at least financially break-even position. If, by any chance, it was ending up profitable, the profit would directly be invested in [Trust and Safety](https://matrix.org/blog/2025/02/building-a-safer-matrix/), or other new programmes which can support the ecosystem. As a reminder, the Foundation is a Community Interest Company, i.e. a limited company which operates to provide a benefit to the community it serves rather than private profit.
+
+
+### What will the freemium offer look like?
+
+The idea is to set some limits for users on the free plans, which would be lifted for users on the premium plans in exchange for an affordable membership. 
+
+**We are still iterating (and will do for a while) on how it looks,** but users can expect limits around media sizes and/or volumes. The goal is to ensure that the most active users participate in covering the costs of the service, in return for the access to a fully encrypted and decentralised open network.
+
+We are limited in scope and design by the fact we need to ship a minimum viable product as soon as possible (we need to reduce costs now) and by not wanting to impose too much development (if any) to Matrix client developers.
+
+Obviously we would have preferred to keep everything free of charge. We will never sell our users’data or cripple  our services with ads, so we need to find ethical sources of revenue. 
+
+### When will the new plans take effect?
+
+The roll-out will happen progressively, starting in the coming weeks and hopefully completing in the summer of 2025. We will start by opening up premium plans to new users only, before progressively migrating all existing accounts to a free plan which will give them the option to upgrade to a premium plan. Users will of course be notified ahead of  their account being migrated.
+
+### Will this work with whatever Matrix client I use? 
+
+Yes. The plan management will be handled via the [My Account](https://account.matrix.org/account/) screens provided by the Matrix Authentication Service (MAS), and notifications to users will be sent in a dedicated room using the [Server Notices](https://spec.matrix.org/v1.14/client-server-api/#server-notices) feature built in to the Matrix protocol – already used by the homeserver to send automatic messages to the user – so should be seamless for every client. 
```

Comment:
grammar nerd strikes again https://dictionary.cambridge.org/dictionary/english/build-in-into
```suggestion
Yes. The plan management will be handled via the [My Account](https://account.matrix.org/account/) screens provided by the Matrix Authentication Service (MAS), and notifications to users will be sent in a dedicated room using the [Server Notices](https://spec.matrix.org/v1.14/client-server-api/#server-notices) feature built into the Matrix protocol – already used by the homeserver to send automatic messages to the user – so should be seamless for every client. 
```